src/main/java/school/lemon/changerequest/java/generics/Pairs/PairUtils.java

```diff
@@ -0,0 +1,58 @@
+package school.lemon.changerequest.java.generics.Pairs;
+
+/**
+ * Created by lbrdev on 21.02.2017.
+ * Project: generics.hw1
+ */
+public final class PairUtils {
+
+    public static <K, V> boolean equals(Pair<K, V> pair1, Pair<K, V> pair2) {
+        boolean flag = false;
+        if (pair1.getKey().equals(pair2.getKey()) && pair1.getValue().equals(pair2.getValue())) {
```

Comment:
It can be much simpler: 
`return pair1.getKey().equals(pair2.getKey()) && pair1.getValue().equals(pair2.getValue())`;